src/index.ts

```diff
@@ -163,6 +163,15 @@ function FlatpickrInstance(
     }
   }
 
+  function isSameTimeAsDefault() {
+    return (
+      self.hourElement &&
+      self.minuteElement &&
+      +self.hourElement.value === self.config.defaultHour &&
```

Comment:
Why the + ?

Reply:
That's the unary operator, this cast the string to a number

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Unary_plus

Reply:
Thank for the education 👌